spec/features/visit_requests/cancel_spec.rb

```diff
@@ -8,8 +9,16 @@
     visit "/events/#{event.slug}"
   end
 
-  it 'removes visit request from database' do
-    expect { click_link 'Cancel attendance'}.to change{VisitRequest.count}.by(-1)
-    expect(page).to have_link 'Attend'
+  it 'changes visit request status to canceled' do
```

Comment:
Test description says that it `changes status`, but we also check for the flash message. This is so called violation of [single expectation test](http://www.betterspecs.org/#single). Feel free to read the whole web site. It has a lot of recommendations
*Action*: Extract `expect(page).to have_content 'Pity.. Hope to see you next time!'` to separate test.
*Warning*: it does not mean that all specs should have only one expectation, for if your code execution is heavy/time consuming - you will add many expectations in 1 `it do block`.